src/service/api/articles.e2e.test.js

```diff
@@ -0,0 +1,214 @@
+'use strict';
+
+const express = require(`express`);
+const request = require(`supertest`);
+
+const articles = require(`./articles`);
+const ArticleService = require(`../data-service/articles`);
+const CommentService = require(`../data-service/comments`);
+const {HTTP_CODE} = require(`../../constants`);
+const mockData = require(`./tests.mocks`);
+
+const createAPI = () => {
+  const app = express();
+  const cloneData = JSON.parse(JSON.stringify(mockData));
+  app.use(express.json());
+  articles(app, new ArticleService(cloneData), new CommentService(cloneData));
+  return app;
+};
+
+describe(`API returns a list of all offers`, () => {
+  const app = createAPI();
+
+  let response;
+
+  beforeAll(async () => {
+    response = await request(app)
+      .get(`/articles`);
+  });
+
+  test(`Status code 200`, () => expect(response.statusCode).toBe(HTTP_CODE.OK));
```

Comment:
```suggestion
  test(`Status code 200`, () => expect(response.statusCode).toBe(HttpCode.OK));
```
Хммм, и тут не так. Видимо это было до меня)